tests/integration/test_integration_all_rust.py

```diff
@@ -69,8 +69,8 @@
 MP_CONNECTION_PORT = 9052
 MP_ROUTER_PORT = 9072
 
-CONNECTION_BINARY = os.environ.get("CONNECTION_BINARY", "autopush_rs")
-CONNECTION_SETTINGS_PREFIX = os.environ.get("CONNECTION_SETTINGS_PREFIX", "autopush__")
+CONNECTION_BINARY = os.environ.get("CONNECTION_BINARY", "autoconnect")
```

Comment:
since we no longer use `autopush_rs`, switching to `autoconnect` for this.

Reply:
Can you kill the `CONNECTION_BINARY/SETTINGS` override in the Makefile's integration-test target along with this?